app/components/bali/side_menu/index.scss

```diff
@@ -0,0 +1,97 @@
+$desktop-side-menu-width: 12rem;
+$side-menu-width: 14rem;
+$navbar-height: 3.75rem;
+$white: hsl(0, 0%, 100%);
+$z-index-side-menu: 5;
+$box-shadow: 0px 3px 18px rgba($black, 0.1), 0 0 0 1px rgba($black, 0.03);
+$fw-500: 500;
+$size-6: 1rem;
+$size-7: 0.875rem;
```

Comment:
We should create a new file with this general size variables

Reply:
Maybe `$white` and `$fw-500` also

Reply:
agree